README.md

```diff
@@ -95,6 +97,62 @@ module.exports = {
 }
 ```
 
+### `Write cache data to redis`
```

Comment:
Could you please try to make this example a bit more generic? e.g `Database Intergation` something like that :)